script/Ciel/Day 9/09_01_CIEL09_2B.txt

```diff
@@ -1071,12 +1242,12 @@ She is absolutely not doing it for the sake of us humans."
 [sha:af5fba97a7f44654fb1d7af823f42918e9514614]{
 -- Page 229, Offset 22722.
 -- 「……なぜ彼女が弱っているのかは判りませんが、もし力が戻れば遠野くんの手を借りる必要なんてなくなります。
-"...I don't know why she's gotten weaker, but if she recovers her power, she won't be needing your help any longer.
+"I don't know why she's weak right now, but if she recovers her power, she won't be needing your help any longer.
 }
 [sha:a2634ce38366931a54da418597e930121a502633]{
 -- Page 229, Offset 22723.
 -- その時、彼女が遠野くんを無事に帰すと思いますか？」
-When that happens, do you think that she'll just let you go?"
+When that happens, do you really think that she'll just let you walk away?"
 }
 [sha:0ebd6b4e290aca46749a5c36b55e53783a6d28a2]{
 -- Page 230, Offset 22724.
```

Comment:
This forward, I'd rather use "truly" instead of "serious" for this series of statements, with serious smudged into the first anyway:
>Ciel is truly serious about this.
She is truly concerned for my life.
She truly despises Arcueid, seeing her as a bloodsucking monster.